sound/soc/sof/intel/hda.h

```diff
@@ -452,6 +456,9 @@ int hda_dsp_core_reset_power_down(struct snd_sof_dev *sdev,
 void hda_dsp_ipc_int_enable(struct snd_sof_dev *sdev);
 void hda_dsp_ipc_int_disable(struct snd_sof_dev *sdev);
 
+int hda_dsp_set_state(struct snd_sof_dev *sdev,
```

Comment:
suggestion:
set_power_state(sdev, dstate, d0_substate).
That way in a future PR we can deprecate CTX_SAVE/RESTORE and use this set_power_state instead

Reply:
OK, thanks.